homework_5/HW5_1_gov_uniq.py

```diff
@@ -0,0 +1,37 @@
+import xml.etree.ElementTree as ET
+filename = 'mondial-3.0.xml'
+path  = 'mondial-3.0.xml'
```

Comment:
These variables aren't used in the program.

The **parse_and_remove** function takes 'mondial-3.0.xml' value as filename and 'country' as path on line 26.